docs/rules/no-container.md

```diff
@@ -0,0 +1,36 @@
+# Disallow the use of `container` methods (no-container)
+
+By using `container` methods like `.querySelector` you may lose a lot of the confidence that the user can really interact with your UI. Also, the test becomes harder to read, and it will break more frequently.
+
+## Rule Details
+
+This rule aims to disallow the use of `container` methods in your tests.
+
+Examples of **incorrect** code for this rule:
+
+```js
+const { container } = render(<Example />);
+const button = container.querySelector('.btn-primary');
+```
+
+```js
+const { container: alias } = render(<Example />);
+const button = alias.querySelector('.btn-primary');
+```
```

Comment:
Can we add some scenarios here?

I can think of 

```js
// using container from screen
const button = screen.container.querySelector('.btn-primary')

// using container from the result from calling render, without destructuring
const view = render(<Example />)
const button = view.container.querySelector('.btn-primary')
```

Reply:
sorry @thebinaryfelix  can we remove this one ?

```ts
const button = screen.container.querySelector('.btn-primary')
```

it was my mistake (2nd time I do this error 😢 ) - `container` is not available in the `screen` object - so this code makes no sense. I am sorry!